app/src/main/java/com/example/spacex/MainActivity.kt

```diff
@@ -40,10 +49,31 @@ class MainActivity : AppCompatActivity() {
         setupActionBarWithNavController(navController, appBarConfiguration)
 
         navView.setupWithNavController(navController)
+
+        createNotificationChannel()
```

Comment:
should not be inside activity, should be as separate object, init in the app level